src/components/InformationDashboard/InfoPreviewModal.jsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react'
+import {
+  Button,
+  Modal,
+  Container,
+  TextField,
+  Card,
+  Grid,
+  CardContent,
+  Box,
+} from '@material-ui/core'
+
+import Information from '../../modules/Information'
+import infoPreview from '../../theme/infoPreviewTheme'
+
+const InfoPreviewModal = ({ informationItem }) => {
+  const classes = infoPreview()
+  const [open, setOpen] = useState(false)
+  const [currentInformation, setCurrentInformation] = useState()
+  const headerMaxLength = 40
+  const descriptionMaxLength = 300
+
+  const getInformation = async () => {
+    let response = await Information.show(informationItem.id)
+    setCurrentInformation(response)
+  }
+
+  const handleOpen = () => {
+    getInformation()
+    setOpen(true)
+  }
+
+  const handleClose = () => {
+    setOpen(false)
+  }
+
+  const handleSubmit = () => {
+    Information.update(informationItem)
+    setOpen(false)
+  }
+  return (
+    <>
+      <Button
+        data-cy='edit-button'
+        type='button'
+        name='edit'
+        variant='contained'
+        color='primary'
+        onClick={() => {
+          handleOpen()
+        }}>
+        Edit
+      </Button>
+      {currentInformation && (
+        <Modal open={open} className={classes.modal}>
+          <Container data-cy='info-container' className={classes.card}>
+            <Card elevation={2} className={classes.fullHeight}>
+              <Grid container direction='row' className={classes.fullHeight}>
+                <Grid item sm={10} xs={11}>
+                  <CardContent className={classes.cardContent}>
+                    <TextField
+                      className={classes.contentField}
+                      label={`Header (max ${headerMaxLength} char.)*`}
+                      data-cy='info-header'
+                      fullWidth
+                      multiline
+                      inputProps={{ maxLength: headerMaxLength }}
+                      defaultValue={currentInformation.header}
+                    />
+                    <TextField
+                      className={classes.contentField}
+                      label={`Description (max ${descriptionMaxLength} char.)*`}
+                      data-cy='info-description'
+                      multiline
+                      fullWidth
+                      inputProps={{ maxLength: descriptionMaxLength }}
+                      defaultValue={currentInformation.description}
+                    />
```

Comment:
add minRows = {2} so they see it is not 1 line here

Reply:
fixed